src/ansys/fluent/core/__init__.py

```diff
@@ -176,3 +179,9 @@ def version_info() -> str:
 
 # Skip password check during rpc execution when Fluent is launched from PyFluent
 LAUNCH_FLUENT_SKIP_PASSWORD_CHECK = False
+
+# `ansys.physicalquantities` will eventually be a separate package. Until then:
+# Inject `physicalquantities` into the `ansys` namespace if it doesn't already exist
+if not hasattr(sys.modules["ansys"], "physicalquantities"):
+    sys.modules["ansys.physicalquantities"] = physicalquantities
+    sys.modules["ansys"].physicalquantities = physicalquantities
```

Comment:
1. Hardcoding module names as strings can lead to typos and makes refactoring harder. We can use constants or variables so that it will improve the error handling as follows::

```
ANSYS_MODULE = "ansys"
QUANTITIES_ATTR = "physicalquantities"
ANSYS_QUANTITIES_MODULE = f"{ANSYS_MODULE }.{QUANTITIES_ATTR }" 

ansys_module = sys.modules.get(ANSYS_MODULE)

if ansys_module is not None and not hasattr(ansys_module , QUANTITIES_ATTR):
    sys.modules[ANSYS_QUANTITIES_MODULE] = QUANTITIES_ATTR 
    setattr(ansys_module, QUANTITIES_ATTR, QUANTITIES_ATTR )
else:
    raise ImportError(f"Module '{ANSYS_MODULE}' not found in sys.modules")
```